pyproject.toml

```diff
@@ -48,31 +48,34 @@ jedi = "^0.18.1"
 darkdetect = ">=0.5.1,<0.9.0"
 better-exceptions = "^0.3.3"
 pydot = "^1.4.2"
-sphinx = { version = ">=4.3,<5.4", optional = true }
+sphinx = { version = ">=4.3,<6.0", optional = true }
 sphinx-copybutton = { version = "^0.5.0", optional = true }
 myst-nb = { version = "^0.17.1", optional = true }
 furo = { version = "^2022", optional = true }
 
 [tool.poetry.group.dev.dependencies]
 pytest = "^7.1"
 pytest-archon = "^0.0"
-pytest-cov = "^3.0"
+pytest-cov = "^4.0.0"
 pytest-mock = "^3.8.2"
 xdoctest = "^1.1.0"
 hypothesis = "^6.54.5"
-sphinx = ">=4.3,<6.0"
 ipython = "^8.5.0"
+sphinx = ">=4.3,<6.0"
 
 [tool.poetry.group.lint.dependencies]
 pre-commit = "^2.20"
 
+[tool.poetry.group.automation.dependencies]
+poethepoet = "^0.17.1"
+
+[tool.poetry.group.translation]
+optional=true
+
```

Comment:
Should the babel stuff be optional? I mean: we have to run it regularly, and it's a bit inconvenient to have to remember to install it.

Reply:
Ya, I think that is a good point. I updated the PR just to be a dependency update.